src/phoenix/server/api/mutations/chat_mutations.py

```diff
@@ -530,6 +536,22 @@ def _serialize_event(event: SpanException) -> dict[str, Any]:
     return {k: (v.isoformat() if isinstance(v, datetime) else v) for k, v in asdict(event).items()}
 
 
+_AnyT = TypeVar("_AnyT")
+
+
+def _get_batches(
+    iterable: Iterable[_AnyT],
+    batch_size: int = 10,
+) -> Iterator[List[_AnyT]]:
+    """Splits an iterable into batches not exceeding a specified size."""
```

Comment:
```suggestion
) -> Iterator[list[_AnyT]]:
```

Reply:
whoops, applied this comment to the wrong line. i meant to use `list` rather than `List` in the type hint.

Reply:
got it. good catch. thx